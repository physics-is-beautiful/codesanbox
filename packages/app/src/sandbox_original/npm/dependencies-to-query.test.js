import dependenciesToQuery from './dependencies-to-query';

it('creates a right query', () => {
  const packages = {
    react: 'latest',
    'react-dom': '15.5.3',
    '@angular/core': '14.0.0',
  };

  expect(dependenciesToQuery(packages)).toMatchSnapshot();
});
