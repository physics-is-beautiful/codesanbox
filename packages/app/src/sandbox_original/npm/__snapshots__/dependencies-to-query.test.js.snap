// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creates a right query 1`] = `"%40angular%2Fcore%4014.0.0+react%40latest+react-dom%4015.5.3"`;
